src/test/requestMocks/config-reqmocks.ts

```diff
@@ -147,11 +147,58 @@ let requestResponseArray: { request: object, response: object }[] = [
       }
     ]
   },
+  {
+    request: {
+      "json":true,
+      "method":"GET",
+      "url":"https://gaia.cruxpay.com/14WfvwxX5G9Nb1C4pq2RHn2bUophFLzrgf/cruxdev_client-config.json"
+    },
```

Comment:
probably this is a duplicate mock. Remove redundant mocked values to avoid confusions.